app1.js

```diff
@@ -13,10 +16,11 @@ function getUserName() {
     document.write(text);
 }
 
+
 function dealInAbsolutes() {
     let userAnswer = prompt('Do you deal in absolutes?');
 
-    if(userAnswer.toLowerCase() === 'no') {
```

Comment:
This could be a constant. That'll make it easier to find with your tools later on and also change the way you might use 'yes' or 'no' values later.

```
const NO_STRING = 'no';
const YES_STRING = 'yes';
```